test/suite/run.py

```diff
@@ -117,6 +117,8 @@ def usage():
   -h      | --help               show this message\n\
   -j N    | --parallel N         run all tests in parallel using N processes\n\
   -l      | --long               run the entire test suite\n\
+  -o N    | --Nth N              randomly sort scenarios to be run, then\n\
```

Comment:
Option here might be better as -N? so you'd do something like run.py -N 20? Could also be -r potentially for random.

Reply:
I agree re matching `-n` for the short version. How about leaving out a short option altogether and calling it `--random-sample`. I'd be happy with either of those choices.